app/search/searchers/controller_searcher.rb

```diff
@@ -56,9 +59,7 @@ class << self
           # on the controller, and then merges in the top-level :page parameter.
           def permitted_controller_params(controller, params_method_name)
             search_specific_params = controller.send(params_method_name)
-            permitted_hash = search_specific_params.to_h
-            permitted_hash[:page] = controller.params[:page] if controller.params.key?(:page)
-            permitted_hash
+            search_specific_params.to_h
```

Comment:
Actually, I don't remember why we merged the top level :page parameter in here before (which you removed). But as the searches still work, I guess we didn't need it. Feel free to close.

Reply:
From the [Pagy common options](https://ddnexus.github.io/pagy/toolbox/paginators/#common-options):

> `page: 3`
> Set it only to force the current :page. (It is set automatically from the request param).

That's why we don't have to explicitly deal with the page (except in tests when we explicitly want to _set_ a page that is not page 1. UI tests should check for us that the correct results for a next page are actually retrieved.

Fixed the tests in 7f81837d4b5d0a